plutarch-test/plutarch-base/Plutarch/PlutusTypeSpec.hs

```diff
@@ -289,23 +288,9 @@ describe "sanity checks" $ do
     it "works" $
  -}
 
-{- |
-  A Sum type, which can be encoded as an Enum
--}
 data AB (s :: S) = A | B
-
-{- |
-  AB is encoded as an Enum, using values of PInteger
-  internally.
--}
-instance PlutusType AB where
-  type PInner AB _ = PInteger
-
-  pcon' A = 0
-  pcon' B = 1
-
-  pmatch' x f =
-    pif (x #== 0) (f A) (f B)
```

Comment:
Why did you remove this?

Reply:
Because it's not a good example anymore! You're never supposed to implement `PlutusType` yourself. In fact, `pcon'` and `pmatch'` aren't even exported anymore I think!